### PR TITLE
古田京平 EX7 解答 Python

### PR DESCRIPTION
**Pull request by tour2st**
# What
- 第7回課題の実装を行いました。
- 実行コマンド
- main.pyファイルをB4Lecture/ex7に配置する
 `$ python main.py --path_to_truth test_truth.csv`

# Result
1回目:0.9866666
result_1_0 986666

2回目:0.9866666
result_2_0 986666

3回目:0.9866666
result_3_0 986666

4回目:0.9866666
result_4_0 986666

5回目:0.99
result_5_0 99

### CODE REVIEW

**Review by kimi-research — COMMENTED** [review]
課題お疲れ様です。
特徴量抽出や構築したモデルがユニークで、高いAccuracyが出ていて良いと思います！
一つだけ気になったのですが、学習のoptimizerをAdamに置き換えてみたりはしましたか？やってなかったら試してみてもいいかもです！

**Comment by kimi-research** [comment]
> from tensorflow.keras.layers import (Conv2D, Dense, Dropout, Flatten, MaxPooling2D) というように複数行で書くとisortが落ちて、 from tensorflow.keras.layers import Conv2D, Dense, Dropout, Flatten, MaxPooling2D というように1行で書くとlinterが落ちるのですがどのように書けばコードチェッカーを通過しますか？ 初歩的な質問ですみません

こちら見落としてました。すみません...たくさんインポートして一行が長くなるからlinterが通らなくなるって感じですかね？
isortの方が通らないのは何故かわからないですが、いっそのこと`from tensorflow.keras import layers`みたいにして、`layers.Dense`みたいな感じで使ってはいかがでしょうか？
自分もあまり詳しくなくてすみません...

File: ex7/k_furuta/main.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+        paths: テストする音声ファイルリスト
+        outputs:
+    Returns:
+        Nothing
+    """
+    with open("result.csv", "w") as f:
+        f.write("path,output\n")
+        assert len(paths) == len(outputs)
+        for path, output in zip(paths, outputs):
+            f.write("{path},{output}\n".format(path=path, output=output))
```
Comment (KateSawada) [inline ex7/k_furuta/main.py:129]:
```suggestion
            f.write(f"{path},{output}\n")
```
f-stringという機能を使うと，こんな感じで文字列中に直接変数を埋め込むことができてシンプルに記述できるのでオススメです

**Comment by tour2st** [comment]
レビューの通りに修正しました！
フィードバックありがとうございます！

**Review by kimi-research — APPROVED** [review]
確認しました！
Approveしておきます。お疲れ様でした！
